client_dashboard/common/src/commonMain/kotlin/org/thechance/common/data/remote/gateway/BaseGateway.kt

```diff
@@ -20,7 +20,7 @@ abstract class BaseGateway {
         try {
             return client.method().body()
         } catch (e: ClientRequestException) {
-            val errorMessages = e.response.body<ServerResponse<*>>().status?.errorMessages
+            val errorMessages = e.response.body<ServerResponse<String>>().status?.errorMessages
```

Comment:
leave todo to next deploy
```suggestion
            //todo: change to ServerResponse<T> in next deploy and test
```